models/calendar.yml

```diff
@@ -359,7 +359,12 @@
     - 9:00 PM:
         type: Talk
         emoji: 🤔
-        title: TBD
+        title: Web Animations
+        description: |
+        With so many different ways to animate the web, it can be hard 
```

Comment:
I have the repo cloned but I can't get it to show any content. I'm seeing a bunch of errors in the console like "Uncaught Error: Cannot find module "./index.md"". Anyway hope this fixed it :(